zendriver/core/browser.py

```diff
@@ -358,6 +364,12 @@ async def start(self) -> Browser:
                 break
 
         if not self.info:
+            stderr = None
+            try :
+                 _, stderr_bytes = await self._process.communicate()
+                 stderr = stderr_bytes.decode()[:1000]
+            except Exception :
+                pass
```

Comment:
is the try/except needed? when would this cause an exception?

if this is to handle `self._process` being `None`, we should check for that explicitly instead of doing try/except

Reply:
also, why are we truncating to first 1000 characters? maybe if anything getting the last 1000 characters would be more useful?

Reply:
about exception handling : communicate is hard method (it use system calls and can raise exception by much reasons), and raise exception in block that already process exception isn't good for me ... in this case we can live with original exception without output (and debug output separatly)
about output : I got first 1000 chars, because I don't expect here much output (we only started chrome here and don't pushed some commands) - it is trivial protection for memory overuse for strange cases (when chrome replaced with something)